src/boat_simulator/boat_simulator/common/constants.py

```diff
@@ -30,6 +31,18 @@ class PhysicsEnginePublisherTopics:
     WIND_SENSORS: str = "mock_wind_sensors"
 
 
+@dataclass
+class BoatProperties:
+    sail_lift_coeffs: Dict[float, float]
+    sail_drag_coeffs: Dict[float, float]
+    sail_areas: Dict[float, float]
+    rudder_drag_coeffs: Dict[float, float]
+    rudder_areas: Dict[float, float]
+    sail_dist: float
+    rudder_dist: float
+    hull_drag_factor: float
```

Comment:
Could you please add comments specifying the units for each property (eg. if it is in meters, degrees, etc.)

Reply:
Added